modules/gitops-release-notes-1-14-1.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-1_{context}"]
+= Release notes for {gitops-title} 1.14.1
+
+{gitops-title} 1.14.1 is now available on {OCP} 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-14.1_{context}"]
+== Errata updates
+
+[id="RHEA-XXXX:NNNN-gitops-1-14-1-security-update-advisory_{context}"]
+=== RHEA-XXXX:NNNN - {gitops-title} 1.14.1 security update advisory
+
+Issued: 2024-XX-YY
+
+The list of security fixes that are included in this release are documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHEA-XXXX:NNNN[RHEA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-14-1_{context}"]
+== Fixed issues
+
+* Before this update, resource limits were set on the operator container as a best practice, which resulted in functional issues on clusters containing a large number of `Secrets` and `ConfigMaps`. This issue arises due to the operator container requiring more memory than the permitted limits. With this update, the resource limits set on the manager container have been removed to minimize the impact on functionality. Efforts are focused to optimize memory consumption for future releases. link:https://issues.redhat.com/browse/GITOPS-5665[GITOPS-5665]
+
+* Before this update, Argo CD could not obtain the appropriate TLS certificate for Helm OCI registries when the URL included a path or port number. With this update, a fix is introduced in upstream Argo CD to correctly parse the URL and return a valid certificate. link:https://issues.redhat.com/browse/GITOPS-5081[GITOPS-5081]
+
+[id="breaking-change-1-14-1_{context}"]
+== Breaking change
+
+[id="additional-configurations-for-cluster-scoped-rollouts-instance-installation_{context}"]
+=== Additional configurations for cluster-scoped rollouts instance installation
+
+* When you upgrade to {gitops-title} v1.14, if you want to create the cluster-scoped rollouts installation outside the default installation namespace, `openshift-gitops-operator`, you must host it in the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable of the `Subscription` resource. The {gitops-title} Operator does not support cluster-scoped rollouts installation if the namespace is not defined in the `CLUSTER_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable.
++
+In previous versions of {gitops-title}, Argo Rollouts used the `NAMESPACE_SCOPED_ARGO_ROLLOUTS_NAMESPACES` environment variable of the `Subscription` resource to check if the cluster-scoped rollouts instance can host a cluster-scoped rollouts installation in the user-defined namespace.
```

Comment:
We can rephrase this line
`if the cluster-scoped rollouts instance can host a cluster-scoped rollouts installation in the user-defined namespace.` => `if user can create a cluster-scoped rollouts instance in the user-defined namespace.`

Reply:
Fixed.